examples/examples.rst

```diff
@@ -9,6 +9,12 @@ Listed below are example jupyter-notebooks
 
 `ICESat-2_DAAC_DataAccess2_Subsetting <https://github.com/icesat2py/icepyx/blob/main/examples/ICESat-2_DAAC_DataAccess2_Subsetting.ipynb>`_
 
-`ICESat-2_DEM_comparison_Colombia_working <https://github.com/icesat2py/icepyx/blob/main/examples/ICESat-2_DEM_comparison_Colombia_working.ipynb>`_
+`Working_with_ICESat-2_Data_Variables <https://github.com/icesat2py/icepyx/blob/development/examples/Working_with_ICESat-2_Data_Variables.ipynb>`_
```

Comment:
Assuming that you want to point to the 'main' branch rather than 'development'?
```suggestion
`Working_with_ICESat-2_Data_Variables <https://github.com/icesat2py/icepyx/blob/main/examples/Working_with_ICESat-2_Data_Variables.ipynb>`_
```